### ISSUE TITLE
Guests not working

### ISSUE THREAD

**Issue by mmattel** [owncloud/guests#191]
Running 10.0.7, upgraded via tar from 10.0.6, php7.0, mysql, Ubuntu 16.04, Client W7x64 / Opera and Edge

Everything is working fine, sharing with created users or via link is working.
The user I am logged on has admin privileges (group admin).
Tested also with another user with admin rights.

Installed guests (did not used before) - no problem, no errors, settings/sharing shows that guests has a whitelist. Watching the youtube video for getting it right https://www.youtube.com/watch?v=L42PBHgqKVI

Whatever I do, on local or external mounts / folders, guests is not working.
``Add as guest user`` is not showing up, as you see in the sceenshot.
I run tail -f on the owncloud log, but there is nothing happening (which is ok)
![image](https://user-images.githubusercontent.com/3321281/36497894-f7352976-173c-11e8-8ed9-a7c764b2db4e.png)

**Comment by cortho** [comment]
I do not see the @ sign in your screenshot, which is mandatory to be able to create a guest. Can you confirm, that guests are not suggested with e.g. mercur@ ?

**Comment by mmattel** [comment]
Aaahhhhh....

So following the video, on minute 2, the guy is **NOT** entereing an @ sign (eMail address) but getting the drop down list opened querying if adding a guest account and an additional window for further details. 
![image](https://user-images.githubusercontent.com/3321281/36532448-0e0e8794-17c1-11e8-86e7-f3f2e0b7aa49.png) 
Now knowing that you need an eMail address entered instead a username, things get clear and work.
The user is created, you can see him in the user screen.

**Suggestion:**
1. Recreate the video to reflect that you need an eMail address entered for getting guests working. The video also shows an additional window popping up which is not longer present/shown in this release...
2. Add a small screenshot/picture in the app description at marketplace making this more clear that you have to enter an eMail address not a username!

Issue can be closed

**Comment by mmattel** [comment]
I know :see_no_evil:
Was focussing on the video...
Reading but not getting what is already stated, optical driven, therefore the suggestion to add a picture.
Thanks for helping :smiley:

**Comment by cortho** [comment]
Indeed, the video is **very outdated**, it even opens the modal for guest user creation, which we purged ages ago. Feel free to open an issue for this.

**Comment by mmattel** [comment]
Where should this issue be opened, here?

**Comment by cortho** [comment]
I'm not sure, @pmaier1 what do you think?

**Comment by pmaier1** [comment]
> Was focussing on the video...

In these regards the video is outdated, yes. We conducted changes afterwards regarding the creation of guests (e.g. removed the modal) and also require a mail address syntax to offer guest creation, both to improve usability. I wrote a blog post on this which you can find here: https://owncloud.org/news/guests-0-5-0-released/

It's not necessary to create a ticket for an updated video. We will create more Insights videos when time permits. I hope that is ok for now.

Thanks for taking the time to report here anyway, highly appreciated!

**Comment by mmattel** [comment]
@pmaier1 wouldn´t it be good to remove the video from youtube (and from the description text includes the one from appstore? This is not a good branding showing things that are outdated...

**Comment by pmaier1** [comment]
> @pmaier1 wouldn´t it be good to remove the video from youtube (and the link from the description text includes the one from appstore? This is not a good branding showing things that are outdated...

Well, the video still explains the functionality and shows how it works which is why I want to leave it as is. We will add a hint in the description that the flow has changed and link to the blog post I mentioned.
